packages/atomic/stencil.config.ts

```diff
@@ -36,7 +39,12 @@ export const config: Config = {
   devServer: {
     reloadStrategy: 'pageReload',
   },
-  plugins: [inlineSvg()],
+  plugins: [
+    inlineSvg(),
+    stencilTailwind({
```

Comment:
I'm wondering what it is about this plugin that makes tailwind work inside the shadow dom without javascript heap-out-of-memory issues. I find the current approach really elegant, but I am worried about running into issues as we use tailwind's different features. For example, it seems like the plugin does not work with responsive classes which would be critical for us: https://github.com/jrowlingson/stencil-tailwind/issues/13

I wonder if there is another solution that has more community support. I found this write-up, could it be an alternative approach?
- https://dev.to/43081j/using-tailwind-at-run-time-with-web-components-47c
- https://github.com/tw-in-js/twind/